projects/packages/masterbar/tests/php/test-class-wpcom-additional-css-manager.php

```diff
@@ -40,15 +40,26 @@ public function set_up() {
 	 * Check if the manager constructs the proper url and copy message.
 	 */
 	public function test_it_generates_proper_url_and_nudge() {
-		$manager = new WPCOM_Additional_CSS_Manager( 'foo.com' );
+		// @phan-suppress-next-line PhanDeprecatedFunction -- Keep using setMethods until we drop PHP 7.0 support.
+		$manager = $this->getMockBuilder( WPCOM_Additional_CSS_Manager::class )
+			->setConstructorArgs( array( 'foo.com' ) )
+			->setMethods( array( 'get_plan' ) )
+			->getMock();
+
+		$manager->method( 'get_plan' )->willReturn(
+			(object) array(
+				'product_name_short' => 'Premium',
+				'path_slug'          => 'premium',
+			)
+		);
 
 		$manager->register_nudge( $this->wp_customize );
 		$this->assertEquals(
 			'/checkout/foo.com/premium',
 			$this->wp_customize->controls()['jetpack_custom_css_control']->cta_url
 		);
 		$this->assertEquals(
-			'Purchase the Explorer plan to<br> activate CSS customization',
+			'Purchase the Premium plan to<br> activate CSS customization',
```

Comment:
Maybe we should create a new test for when the returned plan is `Personal`, that way we won't need to create it once the feature is live and we can just delete this test here.